### PR TITLE
Dev ddcm 45

### PR DESCRIPTION
**Pull request by Gapcroz**

### CODE REVIEW

File: DD/DD.csproj
Comment (Paroxyse) [inline DD/DD.csproj:1]: Remove google forms entries for csproj, they aren't used

File: DD/DD.sln
Comment (Paroxyse) [inline DD/DD.sln:1]: Should this be in version control? @rgr182

File: DD/Middleware/ViewAnalyticsMiddleware.cs
Comment (Paroxyse) [inline DD/Middleware/ViewAnalyticsMiddleware.cs:1]: This shouldn't be in changed files I think, this file is already in stage

File: DD/Views/BackOffice/Analytics.cshtml
Comment (Paroxyse) [inline DD/Views/BackOffice/Analytics.cshtml:1]: dead code

File: DD/Views/LandingPage.cshtml
Comment (Paroxyse) [inline DD/Views/LandingPage.cshtml:1]: don't commit whitespace

File: DD/Views/Partials/homePage/googleForm.cshtml
Comment (Paroxyse) [inline DD/Views/Partials/homePage/googleForm.cshtml:1]: dead code

File: DD/models/ViewAnalytics.cs
Comment (Paroxyse) [inline DD/models/ViewAnalytics.cs:1]: this doesn't correspond to this pr, this file should already be in stage

File: DD/wwwroot/Scripts/googleForm.js
Comment (Paroxyse) [inline DD/wwwroot/Scripts/googleForm.js:1]: dead code

**Review by Paroxyse — CHANGES_REQUESTED** [review]
Remove dead code and sync pr with stage before merging

File: DD/Views/BackOffice/Analytics.cshtml
Comment (Gapcroz) [reply DD/Views/BackOffice/Analytics.cshtml:1]: Erased file.

File: DD/Views/Partials/homePage/googleForm.cshtml
Comment (Gapcroz) [reply DD/Views/Partials/homePage/googleForm.cshtml:1]: erased file.

File: DD/wwwroot/Scripts/googleForm.js
Comment (Gapcroz) [reply DD/wwwroot/Scripts/googleForm.js:1]: erased file.

File: DD/models/ViewAnalytics.cs
Comment (Gapcroz) [reply DD/models/ViewAnalytics.cs:1]: erased file.

**Review by Paroxyse — APPROVED** [review]

**Pull request merged**